app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, old_file_name, new_file_name = command.split()
```

Comment:
what if command is wrong and you will try to unpack less or more elements from a split?